summarize/src/main.rs

```diff
@@ -19,47 +19,47 @@ mod signed_duration;
 
 use query_data::Results;
 
-#[derive(StructOpt, Debug)]
+#[derive(Parser, Debug)]
 struct AggregateOpt {
     files: Vec<PathBuf>,
 }
 
-#[derive(StructOpt, Debug)]
+#[derive(Parser, Debug)]
 struct DiffOpt {
     base: PathBuf,
     change: PathBuf,
 
-    #[structopt(short = "e", long = "exclude")]
+    #[clap(short = 'e', long = "exclude")]
     exclude: Vec<String>,
 
-    #[structopt(long = "json")]
+    #[clap(long = "json")]
     json: bool,
 }
 
-#[derive(StructOpt, Debug)]
+#[derive(Parser, Debug)]
 struct SummarizeOpt {
     file_prefix: PathBuf,
 
     /// Writes the analysis to a json file next to <file_prefix> instead of stdout
-    #[structopt(long = "json")]
+    #[clap(long = "json")]
     json: bool,
 
     /// Filter the output to items whose self-time is greater than this value
-    #[structopt(short = "pa", long = "percent-above", default_value = "0.0")]
+    #[clap(short = 'p', long = "percent-above", default_value = "0.0")]
```

Comment:
minor change in the short flag as Clap do not support shorts that have multiple characters